library/macos/FLEMethodChannel.m

```diff
@@ -14,7 +14,8 @@
 
 #import "FLEMethodChannel.h"
 
-NSString const *FLEMethodNotImplemented = @"notimplemented";
+NSString const *FLEMethodNotImplemented = @"Not Implemented";
```

Comment:
This is a sentinel value used by FLEMethodChannel; its value is irrelevant. (In the iOS implementation, it's actually just a plain NSObject)

Reply:
Since the value doesn't matter, please revert this change.